examples/contract-deployment.example.ts

```diff
@@ -27,4 +27,24 @@ import {BigNumber} from "@ethersproject/bignumber";
      */
     const contract = await contractFactory.deploy(BigNumber.from("10000"), {gasLimit: 3000000});
     console.log(contract.address);
+
+    /**
+     * Example 3. Calling a contract method
+     */
+    const viewMethodCall = await contract.getInternalCounter({gasLimit: 300000});
```

Comment:
If we are going to have examples for the function calls we should rename this file to `contract.example.ts` and not `contract-deployment`